### PR TITLE
Fixing the Zepplin website link

### PR DESCRIPTION
**Pull request by eastlondoner**
https://docs.mesosphere.com/zeppelin/ doesn't work

### CODE REVIEW

File: repo/packages/Z/zeppelin/3/package.json [hunk 1]
```diff
@@ -14,6 +14,6 @@
         "notebook",
         "interactive"
     ],
-    "website": "https://docs.mesosphere.com/zeppelin/",
+    "website": "https://github.com/mesosphere/dcos-zeppelin",
     "postInstallNotes": "DC/OS Zeppelin is being installed!\n\n\tDocumentation: https://docs.mesosphere.com/zeppelin/\n\tIssues: https://docs.mesosphere.com/support/"
```
Comment (ryadav88) [inline repo/packages/Z/zeppelin/3/package.json]: Can you also change the `postInstallNotes` to: 
```DC/OS Zeppelin is being installed!\n\n\tDocumentation: https://github.com/dcos/examples/tree/master/1.8/zeppelin/\n\tIssues: https://dcos.io/community/```

Comment (joerg84) [inline repo/packages/Z/zeppelin/3/package.json]: @eastlondoner Could you take care of the above change? I would be happy to merge it then.

Comment (joerg84) [inline repo/packages/Z/zeppelin/3/package.json]: Ok I will fix this in another commit.

**Comment by joerg84** [comment]
Fixed the requested change with https://github.com/mesosphere/universe/pull/941

**Pull request merged**